### DISCUSSION TITLE
:tada: but not sure what to call this

### DISCUSSION THREAD

**Discussion by jgarza9788** [Schneegans/Burn-My-Windows#465] [Ideas]
if you want the code early it's here
https://github.com/jgarza9788/Burn-My-Windows/tree/spherize

https://github.com/user-attachments/assets/b4e12d1a-404d-40eb-9d21-8a32aa3017ab

**Comment by Schneegans** [comment]
I think this could be interesting with some sort of wavy ripple effect! Like a drop of water falling into the window...

**Comment by Schneegans** [comment]
Something like this: https://www.shadertoy.com/view/llj3Dz
